packages/lesswrong/components/common/Header.tsx

```diff
@@ -196,26 +197,26 @@ class Header extends PureComponent<HeaderProps,HeaderState> {
   // Called by SearchBar.
   setSearchOpen = (isOpen) => {
     const { captureEvent } = this.props
-    if (isOpen) { captureEvent("searchToggle", {"open": isOpen}) }
+    if (isOpen) { captureEvent("searchToggle", { "open": isOpen }) }
     this.setState({
       searchOpen: isOpen
     });
   }
 
   renderNavigationMenuButton = () => {
-    const {standaloneNavigationPresent, toggleStandaloneNavigation, classes, toc} = this.props
+    const { standaloneNavigationPresent, toggleStandaloneNavigation, classes, toc } = this.props
     const { unFixed } = this.state
     return <React.Fragment>
       <IconButton
         className={classNames(
           classes.menuButton,
-          {[classes.hideOnDesktop]: standaloneNavigationPresent && unFixed}
+          { [classes.hideOnDesktop]: standaloneNavigationPresent && unFixed }
         )}
         color="inherit"
         aria-label="Menu"
-        onClick={()=>this.setNavigationOpen(true)}
+        onClick={() => this.setNavigationOpen(true)}
```

Comment:
I think you have your editor misconfigured to run an automatic code formatter over files. This is something you want to avoid, because it creates lots of merge conflicts.

Reply:
Yup, we figured that out, and I ran of steam for the day before we had time to fix it. I'm not even sure this PR will be merged anyways so wasn't that worried about it locally but yeah we'd want to fix it.